templates/components/timeline.html.twig

```diff
@@ -11,15 +11,15 @@
                 {% endif %}
                 {% if true %}
                     <div class="list-timeline-content">
-                        {% if item.time is defined %}
+                        {% if item.time is defined and item.time is not null %}
                             <div class="list-timeline-time">{{ item.time }}</div>
                         {% endif %}
-                        {% if item.title is defined %}
+                        {% if item.title is defined and item.title is not null %}
                             <p class="list-timeline-title">{{ item.title }}</p>
                         {% endif %}
-                        {% if item.text is defined %}
+                        {% if item.text is defined and item.text is not null %}
                             <p class="text-muted">{{ item.text|raw }}</p>
```

Comment:
I know you did not change it, but while looking at your changes I saw that line as well.
What do you think? In my opinion the |raw should be removed here. It is dangerous if used in a context where the user defined the text.

Reply:
imho: `item.text` should be `plain text`. otherwise one could use `item.textHtml`

What I did not check is whether the above changes 'destroy' the template if you do not have eg. `item.time` set.
I will check on that later.

Reply:
➕ 1 to remove the `raw` with the simple `text` option.